### PR TITLE
Fix ABI spec.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/abi-spec.rst [hunk 1]
```diff
@@ -58,15 +58,15 @@ The following elementary types exist:
 
 The following (fixed-size) array type exists:
 
-- ``<type>[M]``: a fixed-length array of the given fixed-length type.
+- ``<type>[M]``: a fixed-length array of N elements, ``N > 0``, of the given type.
```
Comment (axic) [inline docs/abi-spec.rst]: The left refers to `M`.

Comment (chriseth) [reply docs/abi-spec.rst]: argh...

**Comment by axic** [comment]
@chriseth fixed and added a clarification on the shorthand in selector.

**Review by axic — APPROVED** [review]

**Comment by axic** [comment]
@chriseth please review

**Pull request merged**